PC02_Graffiti_EPPERSON.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Max Epperson
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("lightsteelblue1")
+panel.bgpic(image)
+
+#=======Add your code here======
+home()
+showturtle()
+up()
+clear()
+goto(120,-90)
+clear()
+
+color(168,204,215) #piss jar base
+width(5)
+down()
+fillcolor(225,225,20)
+begin_fill()
+left(90)
+forward(50)
+right(45)
+goto(130,-30)
+right(45)
+forward(100)
+right(45)
+goto(240,-40)
+right(45)
+forward(150)
+right(45)
+goto(230, -200)
+right(45)
+forward(100)
+right(45)
+goto(120,-190)
+right(45)
+forward(100)
+end_fill()
+
+up()            #glass details
+goto(130, -190)
+down()
+right(90)
+forward(100)
+up()
+goto(130,-40)
+down()
+forward(100)
+
```

Comment:
Add dot notation, define variables, and soft code values
```suggestion
#variable for RGB value of black
black = (0,0,0)

#Creates the black base of the jar lid
jarTurt.up()       
jarTurt.goto(130,-25)
jarTurt.color(black)
jarTurt.fillcolor(black)
jarTurt.down()
jarTurt.begin_fill()
jarTurt.forward(Longside+50)
jarTurt.left(Lrotation)
jarTurt.forward(20)
jarTurt.left(Lrotation)
jarTurt.forward(Longside)
jarTurt.left(Lrotation)
jarTurt.forward(20)
jarTurt.end_fill()
```